number_trivia_app/lib/features/number_trivia/domain/usecases/get_concrete_number_trivia.dart

```diff
@@ -0,0 +1,17 @@
+import 'package:dartz/dartz.dart';
+import 'package:number_trivia_app/core/error/failures.dart';
+import 'package:number_trivia_app/features/number_trivia/domain/entities/number_trivia.dart';
+import 'package:meta/meta.dart';
+import '../repositories/number_trivia_repository.dart';
+
+class GetConcreteNumberTrivia {
+  final NumberTriviaRepository repository;
+
+  GetConcreteNumberTrivia(this.repository);
+
+  Future<Either<Failure, NumberTrivia>> execute({
+    @required int number,
```

Comment:
Do you like to create the methods with parameters on it's own line?
We can discuss offline